src/controllers/controller.line.js

```diff
@@ -86,8 +88,8 @@ module.exports = DatasetController.extend({
 		var dataset = me.getDataset();
 		var datasetIndex = me.index;
 		var value = dataset.data[index];
-		var yScale = me.getScaleForId(meta.yAxisID);
-		var xScale = me.getScaleForId(meta.xAxisID);
+		var xScale = me._xScale;
+		var yScale = me._yScale;
```

Comment:
It looks like these are not worth caching locally here (only used twice). If my math is right, same rule of 4 uses applies here, before caching is beneficial to minimization.

Reply:
These save 8 bytes, so it is beneficial :)

Reply:
Right, didn't think of these as properties, due to lack of coffee :)